ros2/nightly/nightly/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN echo "deb http://packages.ros.org/ros2-testing/ubuntu `lsb_release -sc` main
 # install bootstrap tools
 RUN apt-get update && apt-get install --no-install-recommends -y \
     git \
+    python3-colcon-common-extensions \
```

Comment:
This should be safe to add back now :+1:

From https://github.com/osrf/docker_images/pull/272#issuecomment-498785604 we may want to keep installing from pip to avoid version incompatibilities between pytest packages from apt and pip

Reply:
Uh, do you want me to revert this change, or update the images.yaml.em so this is installed using pip so the running the template doesn't change it back to apt?

Reply:
hmm I'm not sure what would be the result if we add it to the `images.yaml.em` I guess it would be installed from both apt and pip. I would expect the pip version would take precedence during the build/test invocation but would need to be tested to be sure.

But I agree that the manual override is more of a hack and a cleaner way of doing it is desirable.